Sources/DangerShellExecutor/ShellExecutor.swift

```diff
@@ -77,7 +77,8 @@ public struct ShellExecutor: ShellExecuting {
 
             task.waitUntilExit()
 
-            return String(data: data, encoding: .utf8)!.trimmingCharacters(in: .whitespacesAndNewlines)
+            let result = String(data: data, encoding: .utf8) ?? ""
+            return result.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
Nit

```suggestion
            let result = String(data: data, encoding: .utf8).map { $0.trimmingCharacters(in: .whitespacesAndNewlines) }  ?? ""
            return result
```

So you do the trimming only if not nil

Reply:
fixed